packages/cosmos_ui_components/lib/components/cosmos_token_avatar.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+
+class CosmosTokenAvatar extends StatelessWidget {
+  final String text;
+
+  const CosmosTokenAvatar({Key? key, required this.text}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 32,
+      child: CircleAvatar(
+        backgroundColor: Colors.grey[300],
```

Comment:
we should always use colors from the theme, otherwise we will lack the dark mode/ light mode support and have to pinpoint all those places in code to update it

Reply:
Updated